service/src/main/resources/db/changesets/20220131_resource_type.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+- changeSet:
+    id: resource type fixup
+    author: ddietterich
+    changes:
+    - renameColumn:
+        tableName: resource
+        newColumnName: cloud_resource_type
+        oldColumnName: resource_type
+        remarks: The type of cloud resource, for example, GCS_BUCKET.
+    - addColumn:
+        tableName: resource
+        columns:
+        - column:
+            name: resource_type
```

Comment:
It's confusing & scary to me to reuse an old column name. I could cause unpredictable results, e.g. if someone had a saved query that grabbed `resource_type`.

Reply:
I'm not seeing the unpredictable results, but it isn't necessary to take the risk. I'll make a new name.

The target end state is to remove the cloud_resource_type and stewardship_type columns.
In between, I need to run an update that will properly populate the (currently named) resource_type column.